sdk/storage/Azure.Storage.Blobs/src/BlobServiceClient.cs

```diff
@@ -1667,6 +1665,156 @@ public BlobServiceClient(Uri serviceUri, TokenCredential credential, BlobClientO
         }
         #endregion UndeleteBlobContainer
 
+        #region RenameBlobContainer
+        /// <summary>
+        /// Renames an existing Blob Container.
+        /// </summary>
+        /// <param name="destinationContainerName">
+        /// The new name of the Blob Container.
+        /// </param>
+        /// <param name="sourceContainerName">
+        /// The name of the source container.
+        /// </param>
+        /// <param name="sourceConditions">
+        /// Optional <see cref="BlobRequestConditions"/> to add
+        /// conditions on the renaming of this container.
```

Comment:
nit: "Optional BlobRequestConditions that source has to meet." ? or something like that (current is kind of weird).

Reply:
This matches our documentation elsewhere for access conditions.

Reply:
That doesn't mean it's great.

Reply:
fixed.